fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/AggregateIndexMatchCandidate.java

```diff
@@ -364,10 +364,16 @@ public List<RecordType> getQueriedRecordTypes() {
         return recordTypes;
     }
 
+    protected int getGroupingCount() {
+        return IndexTypes.BITMAP_VALUE.equals(index.getType())
```

Comment:
please introduce something like:

```
final int keyExpressionGroupingCount = ((GroupingKeyExpression)index.getRootExpression()).getGroupingCount();
```

as a local variable and then use it in the `?`.

Reply:
👍